domains/certificates/Query.API/API/MasterDataService/MockMasterDataService.cs

```diff
@@ -58,7 +58,8 @@ private async Task<string> GetMeteringPointOwner(MasterDataMockInput masterDataM
 
         var meteringPointOwner = await clientFactory.CreateClient().GetUuidForCompany(cvr);
 
-        cvrToMeteringPointOwner.TryAdd(cvr, meteringPointOwner);
+        if (!string.IsNullOrWhiteSpace(meteringPointOwner))
```

Comment:
Shouldnt we have a test to catch this?

Reply:
Since this is a mock implementation of the master data service and not the real implementation we decided not to add tests. However, the complexity of the mock implementation has increased since we started, which means we should revise our initial decision of no tests. 
I suggest this as a two step: Step 1: Get this bugfix out. Step 2: Add unit tests to MockMasterDataService.